include/amp_short_vectors.h

```diff
@@ -3,14 +3,23 @@
 #ifndef _AMP_SHORT_VECTORS_H
 #define _AMP_SHORT_VECTORS_H
 
+#include <cstddef>
+#include <type_traits>
+
 namespace Concurrency
 {
 namespace graphics
 {
 
 #define __CPU_GPU__ restrict(cpu, amp)
 
+#if 1
+#include <cstddef>
+#include <type_traits>
+#include "hc_short_vector.inl"
+#else
 #include "kalmar_short_vectors.inl"

```

Comment:
Yes, there will be a 1-2 follow up PR for this.  I'm planning to add a few more tests, add half precision support and remove the old implementation.